static_src/stores/activity_store.js

```diff
@@ -60,21 +60,33 @@ class ActivityStore extends BaseStore {
     super();
     this._data = new Immutable.List();
     this.subscribe(() => this._registerToActions.bind(this));
+    this.eventsFetched = false;
```

Comment:
Minor quibble, but it seems that we've been using `_` prefixed variable names for things that only the store should care about. Should these variables follow that convention or are they intended for public use?